ablator/modules/loggers/main.py

```diff
@@ -217,17 +344,55 @@ def checkpoint(
             self._update_metadata()
 
     def clean_checkpoints(self, keep_n_checkpoints: int):
+        """
+        Clean up checkpoints and keep only the specified number of checkpoints.
+
+        Parameters
+        ----------
+        keep_n_checkpoints : int
+            Number of checkpoints to keep.
+        """
         if self.model_dir is None:
             return
         for chkpt_dir in self.CHKPT_DIR_VALUES:
             dir_path = self.model_dir.joinpath(chkpt_dir)
             futils.clean_checkpoints(dir_path, keep_n_checkpoints)
 
     def info(self, *args, **kwargs):
+        """
+        Log an info message using the logger.
+
+        Parameters
+        ----------
+        *args
+            Positional arguments passed to the logger's info method.
+        **kwargs
+            Keyword arguments passed to the logger's info method.
+        """
         self.logger.info(*args, **kwargs)
 
     def warn(self, *args, **kwargs):
+        """
+        Log a warning message using the logger.
+
+        Parameters
+        ----------
+        *args
+            Positional arguments passed to the logger's warn method.
+        **kwargs
+            Keyword arguments passed to the logger's warn method.
+        """
         self.logger.warn(*args, **kwargs)
 
     def error(self, *args, **kwargs):
+        """
+        Log an error message using the logger.
```

Comment:
I think we should clarify that this logs info to files and to console, not tensorboard